lib/modules/manager/vendir/extract.ts

```diff
@@ -0,0 +1,84 @@
+import is from '@sindresorhus/is';
+import { parseSingleYaml } from '../../../util/yaml';
+import { logger } from '../../../logger';
+import { HelmDatasource } from '../../datasource/helm';
+import type { PackageDependency, PackageFileContent } from '../types';
+import type { HelmChart, Vendir } from './types';
+import { isHelmChart } from './utils';
+
+// TODO: Add support for other vendir types (like git tags, github releases, etc.)
```

Comment:
There are a bunch of TODO's in this file. Do you want to fix these TODO's in _this_ PR, or _later_?

Reply:
The tests being fixed will happen in this PR. The other ones will come in a future PR. I was just tracking the future work needed for vendir

Reply:
Thank you for the explanation. 😄

Reply:
@HonkingGoose Resolved most of the TODOs. There's one left around supporting other datatypes that vendir supports. That can come in separate PRs as users need them.